### PR TITLE
Add xfail list specific for 202012 branch

### PR DESCRIPTION
**Pull request by StormLiangMS**
### Description of PR

Summary:
In order to surface new regression on 202012 branch quickly, put all known issues to the xfail list and track them by github issues. 

Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?
In order to surface new regression on 202012 branch quickly, put all known issues to the xfail list and track them by github issues.

### CODE REVIEW

**Review by wangxin — APPROVED** [review]

File: tests/common/plugins/conditional_mark/tests_mark_conditions_202012.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+##############################################################
+#####  This file is dedicated for 202012 branch issues   #####
+#####  sonic-mgmt issue should be filed and linked here  #####
+##############################################################
+
+dualtor/test_ipinip.py::test_decap_standby_tor
```
Comment (ZhaohuiS) [inline tests/common/plugins/conditional_mark/tests_mark_conditions_202012.yaml:6]: @StormLiangMS All cases are missing ":" at the end of line.

File: tests/common/plugins/conditional_mark/tests_mark_conditions_202012.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    reason: "test issue or image issue, to be RCA'ed"
+    conditions:
+      - "release in ['202012']"
+
+platform_tests/test_advanced_reboot.py
+  xfail:
+    reason: "test issue or image issue, to be RCA'ed"
+    conditions:
+      - "release in ['202012']"
+      - "'dualtor' in topo_name"
```
Comment (ZhaohuiS) [inline tests/common/plugins/conditional_mark/tests_mark_conditions_202012.yaml:66]: @StormLiangMS For this case, I found it failed on both dualtor and singal tor. Should we remove this dualtor condition?

File: tests/common/plugins/conditional_mark/tests_mark_conditions_202012.yaml (hunk 1 above)
Comment (StormLiangMS) [reply tests/common/plugins/conditional_mark/tests_mark_conditions_202012.yaml:6]: good catch, will raise one more PR to address this.

File: tests/common/plugins/conditional_mark/tests_mark_conditions_202012.yaml (hunk 2 above)
Comment (StormLiangMS) [reply tests/common/plugins/conditional_mark/tests_mark_conditions_202012.yaml:66]: sure.

**Pull request merged**